README.md

```diff
@@ -20,9 +20,27 @@ installed from npm.
 
 Overview 
 --------
-
 ReactDataGrid is an advanced JavaScript spreadsheet component built using React
 
+Exported Modules
```

Comment:
I've updated it. can you verify the instructions are clear / correct, esp on migration